src/components/Templates/DefaultSlideTemplate.vue

```diff
@@ -18,77 +15,64 @@
         lg6>
         <v-card
           color="transparent"
-          flat
-          height="100px">
+          flat>
           <v-card-title
             primary
             class="title">
-            <h2 class="display-3">{{ title }}</h2>
+            <h2 class="display-3">{{ slideView.title.content }}</h2>
           </v-card-title>
           <v-card-title
             primary
             class="title">
-            <h2 class="display-2">{{ time }}</h2>
+            <h2 class="display-2">{{ slideView.date.content }}, {{ slideView.time.content }} </h2>
```

Comment:
I'm assuming that this is something build in to Vue? Or was the `date` and `time` functionality something that we needed to create?

Reply:
Yes, the date and time must be displayed on the slide itself for an event.